tests/baselines/reference/exportsAndImports1-es6.js

```diff
@@ -0,0 +1,73 @@
+//// [tests/cases/conformance/es6/modules/exportsAndImports1-es6.ts] ////
+
+//// [t1.ts]
+
+var v = 1;
+function f() { }
+class C {
+}
+interface I {
+}
+enum E {
+    A, B, C
+}
+const enum D {
+    A, B, C
+}
+module M {
+    export var x;
+}
+module N {
+    export interface I {
+    }
+}
+type T = number;
+import a = M.x;
+
```

Comment:
Add in a reassignment to `v` (ie, `v = 10`) here. Observe that we emit `v = 10` but no corresponding `exports.v = v`. I'm not sure lifting all the exports from where the export statement is to wherever the variable's declaration is is correct for this export form.